MeshLib/Elements/FaceRule.cpp

```diff
@@ -15,19 +15,29 @@
 
 namespace MeshLib
 {
-
 bool FaceRule::testElementNodeOrder(const Element* e)
 {
-    MathLib::Vector3 up_vec (0,0,1);
-    return (MathLib::scalarProduct(getSurfaceNormal(e), up_vec) < 0) ? true : false;
+    MathLib::Vector3 const z_unit_vector(0, 0, 1);
+    return MathLib::scalarProduct(getSurfaceNormal(e), z_unit_vector) < 0;
```

Comment:
Wouldn't it be more efficient to just check the sign of the third component of `getSurfaceNormal(e)`?